pymilvus/client/abstract.py

```diff
@@ -184,13 +184,19 @@ def id(self):
 
     @property
     def fields(self):
-        raise NotImplementedError('TODO: support field in Hits')
+        fields = []
+        for k, v in self._row_data.items():
+            fields.append(k)
+        return fields
 
     def get(self, field):
         return self.value_of_field(field)
 
     def value_of_field(self, field):
-        raise NotImplementedError('TODO: support field in Hits')
+        if field in self._row_data:
```

Comment:
Do the fields contain the primary field?

Reply:
yes